### PR TITLE
fix: Add buildng instructions and fixed JUnit dependency, closes #118

### LINKED ISSUE #118: add build and run instructions to readme
as per the TA's comment.

Is this just: how to install Java, open the project in IDE and run Main.java?
Or is this supposed to compile down to a binary?

As I'm not so experienced with Java, would be ace if someone just shoots me the details if it's any more complicated than the above. Then I can add them into the documentation, as it is my domain and you guys have done so much on the code side. 
Of course, if someone else just wants to make a branch and do it themselves it's of course totally cool. No territoriality here 😉

### PR DESCRIPTION
**Pull request by andersblomqvist**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -284,3 +285,21 @@ given our lack of previous experience in such a scenario. For this we are proud.
...
+
+For building and running this project the requirements are:
+
+- [Maven v3.8.6](https://maven.apache.org/docs/3.8.6/release-notes.html)
+- Java 19+
+
+### How to build and run
+
+1. Clone this repository.
+2. Import project as a Maven project with your desired IDE.
```
Comment (tjex) [inline README.md:298]: describe what is meant my 'a Maven project' or explain how to do it / how it differs from just opening a folder in the IDE like normal

File: README.md [hunk 2]
```diff
@@ -284,3 +285,21 @@ given our lack of previous experience in such a scenario. For this we are proud.
...
+3. Run `Main.java`.
+
+Or by manually running with maven:
+
+1. Compile with `mvn compile`
```
Comment (tjex) [inline README.md:303]: steps before step 1: 
- clone repo
- cd to root `cd decide-g26`

**Review by tjex — CHANGES_REQUESTED** [review]
the build worked for me with the instructions, but added a few to make it clearer.
also some more detail needed in the IDE step.

File: README.md (hunk 2 above)
Comment (tjex) [reply README.md:303]: Or via your terminal:

1. Clone this repository.
2. Enter repository root `cd decide-g26`
3. Compile with `mvn compile`
3. Run the program `mvn exec:java -Dexec.mainClass=com.group26.Main`
4. Test the program `mvn test`

**Comment by tjex** [comment]
Also `target/` directory should be included in the `gitignore` as it's created by Maven and is user dependant.

File: README.md (hunk 1 above)
Comment (andersblomqvist) [reply README.md:298]: I mean, thought an IDE you either choose "gradle", "maven", or anything else. I don't know what else to write.

File: README.md (hunk 2 above)
Comment (andersblomqvist) [reply README.md:303]: Good change.

File: README.md [hunk 3]
```diff
@@ -284,3 +285,21 @@ given our lack of previous experience in such a scenario. For this we are proud.
...
+2. Run the program `mvn exec:java -Dexec.mainClass=com.group26.Main`
```
Comment (Darpos) [inline README.md]: mvn exec:java -D"exec.mainClass"="com.group26.Main" instead for powershell :)

**Review by Darpos — APPROVED** [review]
Nice one!

**Review by tjex — APPROVED** [review]
Is the sytax for gitignore `/folder/`? Thought it was `folder/`
If you're sure it's good, then all good.

**Pull request merged**